components/browser/domains/src/test/java/mozilla/components/browser/domains/DomainAutoCompleteProviderTest.kt

```diff
@@ -52,23 +74,23 @@ class DomainAutoCompleteProviderTest {
     @Test
     fun testAutocompletionWithCustomDomains() {
         val domains = listOf("facebook.com", "google.com", "mozilla.org")
-        val customDomains = listOf("gap.com", "fanfiction.com", "mobile.de")
+        val customDomains = listOf("gap.com", "www.fanfiction.com", "https://mobile.de")
 
         val provider = DomainAutoCompleteProvider()
         provider.initialize(RuntimeEnvironment.application, true, true, false)
         provider.onDomainsLoaded(domains, customDomains)
 
-        assertCompletion(provider, "f", AutocompleteSource.CUSTOM_LIST, customDomains.size, "fanfiction.com")
-        assertCompletion(provider, "fa", AutocompleteSource.CUSTOM_LIST, customDomains.size, "fanfiction.com")
+        assertCompletion(provider, "f", AutocompleteSource.CUSTOM_LIST, customDomains.size, "fanfiction.com", hasWww = true)
+        assertCompletion(provider, "fa", AutocompleteSource.CUSTOM_LIST, customDomains.size, "fanfiction.com", hasWww = true)
```

Comment:
\[detekt\] INFO: \[style/MaxLineLength\] Line detected that is longer than the defined maximum line length in the code style\.